src/backend/templates/rooms.hbs

```diff
@@ -120,14 +120,14 @@
 		             	    </p>
 		             	  {{/if}}
 		             	{{/speakers_list}}
-		             	<ul style="padding:0;font-size:15px;">
+                    <a href="javascript:;"><ul style="padding:0;font-size:15px;">
```

Comment:
@utkarshnath Writing `<a href="javascript:;">`  is a bad practice
Check this http://stackoverflow.com/questions/7755088/what-does-href-expression-a-href-javascript-a-do 
You can return false by calling a dummy function.